package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@rdohms/gh-labeller",
+  "version": "0.1.0",
+  "description": "CLI Command that creates GitHub Labels from a set of configuration",
+  "main": "cli.js",
+  "license": "MIT",
+  "devDependencies": {
+    "eslint": "^4.19.1",
+    "eslint-config-prettier": "^3.3.0",
+    "eslint-plugin-import": "^2.12.0",
```

Comment:
Are we using this plugin? If not, we can remove it!

Reply:
not sure to be honest, copied this one over from our old code

Reply:
yeah confirmed we do not need it